package com.company.certification.reto.exceptions;

public class ExpectedResultErr extends AssertionError {
    public static final String EXPECTED_RESULT = "The expected result is not successful";
}
